### PR TITLE
add sources, fix and remove

### LINKED ISSUE #255: EnAreasManga : ( no page found)
### Source information

EnAreasManga

### Steps to reproduce

Access to the source
Choose the manha or chapter
Error: There are no pages

### Kotatsu version

6.1

### Android version

Android 13

### Other details

Pictures of the problem before when this problem existed 
Screenshot_٢٠٢٣٠٩١٢-١٧١٩٠٧
Screenshot_٢٠٢٣٠٩١٢-١٧١٩٢٧

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.

### LINKED ISSUE #256: Add Source ksk.moe
### Source name

ksk.moe

### Source link

https://ksk.moe/

### Language

English

### Other details

Adding Hentai manga Source - https://ksk.moe/

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have checked that the source does not already exist on the app.
- [X] I have checked that the source does not already exist by searching the [GitHub repository](https://github.com/KotatsuApp/kotatsu-parsers) and verified it does not appear in the code base.

### LINKED ISSUE #258: OzulShojo : (Source request)
### Source name

OzulShojo

### Source link

https://ozulshojo.com

### Language

Arabic

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have checked that the source does not already exist on the app.
- [X] I have checked that the source does not already exist by searching the [GitHub repository](https://github.com/KotatsuApp/kotatsu-parsers) and verified it does not appear in the code base.

### LINKED ISSUE #260: OzulScans :(Source request)
### Source name

OzulScans

### Source link

https://ozulscansen.com/

### Language

English

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have checked that the source does not already exist on the app.
- [X] I have checked that the source does not already exist by searching the [GitHub repository](https://github.com/KotatsuApp/kotatsu-parsers) and verified it does not appear in the code base.

### PR DESCRIPTION
**Pull request by davvarrr**
New source :

yugenmangas.net  
reaperscans.net  
po2scans.com  
lire-scan.me  
inarimanga.com  
mangastarz.com  
creepyscans.com  
jpscan-vf.net  
manga168.com  
www.tarotscans.com  
yaoitr.com  
ozulscans.xyz
ozulshojo.com -> close #258 
mangaesp.co  
hentais.sssscanlator.com  
ozulscansen.com -> close #260 
fmteam.fr  
manhwaland.lat
dianxiatrads.com
mhscans.com
webtoonevreni.net
nvmanga.com
hanman18.com
m.manganelo.com
likemanga.io
ksk.moe ( but low-res pages ) -> close #256 
manga1k.com
shadowxmanga.com
mangashiro.me

fix :
Mangaid -> name on update page

legionscans.com move le to cerberuseries.xyz and change template

Murim Scan

move mangalike.org

en-aresmanga.com -> close #255 

perf-scan.fr -> move to heancms

lugnica-scans.com  
( there's probably a good part of the json that needs to be revised/optimized, the chapter date part - I can't figure out how you want to set a timestamp.  
I added this 'Possible improvement here' to better understand where it is )

url :

moondaisyscans.com -> moondaisyscans.biz  
hiperdex.com -> hiperdex.xyz  
doujindesu.rip -> doujindesu.cfd  
manhwadesu.top -> manhwadesu.one  
teamxnovel.com -> team1x12.com
goldenmangas.top -> goldenmanga.top
anigliscans.com -> anigliscans.xyz
asurascanstr.com -> armoniscans.com
nightscans.org -> nightscans.net
swatmanga.co -> stmanga.me

remove sources :
mirrordesu.ink
1stkissmanga.me  ( redirect to likemanga.io )
Peach Scan

### CODE REVIEW

**Comment by davvarrr** [comment]
For lugnica-scans.com I have trouble with the json if you had a better solution you could change the code.
and i don't know if we can add to .tryParse() the possibility to put a Timestamp ( or another method )

File: src/main/kotlin/org/koitharu/kotatsu/parsers/site/en/KskMoe.kt [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+		return doc.requireElementById("galleries").select("article").map { div ->
+			val href = div.selectFirstOrThrow("a").attrAsAbsoluteUrl("href")
+			Manga(
+				id = generateUid(href),
+				title = div.selectLastOrThrow("h3 span").text(),
+				altTitle = null,
+				url = href,
+				publicUrl = href.toAbsoluteUrl(domain),
+				rating = RATING_UNKNOWN,
+				isNsfw = false,
```
Comment (Koitharu) [inline src/main/kotlin/org/koitharu/kotatsu/parsers/site/en/KskMoe.kt:75]: Why `false` if the source type is "Hentai"?

File: src/main/kotlin/org/koitharu/kotatsu/parsers/site/en/KskMoe.kt [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+		} else {
+			webClient.httpGet("https://${domain}/tags/page/$page").parseHtml().body()
+				.getElementById("tags")
+		}
+		return root?.parseTags().orEmpty()
+	}
+
+	private fun Element.parseTags() = select("section.tags div a").mapToSet { a ->
+		MangaTag(
+			key = a.attr("href").substringAfter("/tags/"),
```
Comment (Koitharu) [inline src/main/kotlin/org/koitharu/kotatsu/parsers/site/en/KskMoe.kt:112]:
```suggestion
			key = a.attr("href").substringAfterLast("/tags/"),
```

File: src/main/kotlin/org/koitharu/kotatsu/parsers/site/heancmsalt/HeanCmsAlt.kt [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+	}
+
+	override suspend fun getTags(): Set<MangaTag> = emptySet()
+
+	protected open val selectDesc = "div.description-container"
+	protected open val selectAlt = "div.series-alternative-names"
+	protected open val selectChapter = "ul.MuiList-root a"
+	protected open val selectChapterTitle = "div.MuiListItemText-multiline span"
+	protected open val selectChapterDate = "div.MuiListItemText-multiline p"
+	override suspend fun getDetails(manga: Manga): Manga = coroutineScope {
```
Comment (Koitharu) [inline src/main/kotlin/org/koitharu/kotatsu/parsers/site/heancmsalt/HeanCmsAlt.kt:85]: Is nested `coroutineScope` creation necessary here?

**Comment by garvongithub** [comment]
Ksk images are so low res that they are unreadable. It's so bad that it's not even worth keeping.

**Comment by davvarrr** [comment]
It was already said on the discord, but before adding it I asked the dev if I could add it or not because I couldn't get the high quality images and we concluded that we would add the source like that and if someone could add the high quality images he could do it.
For the moment I don't have the ability to add them so I'm waiting for help.

**Pull request merged**